apps/voting/app/src/components/VotePanelContent.js

```diff
@@ -1,37 +1,63 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import Blockies from 'react-blockies'
 import {
   Button,
   Info,
+  SafeLink,
   SidePanelSplit,
   SidePanelSeparator,
   Countdown,
   Text,
   theme,
 } from '@aragon/ui'
-import { VOTE_ABSENT, VOTE_NAY, VOTE_YEA } from '../vote-types'
+import provideNetwork from '../utils/provideNetwork'
+import {
+  voteStateFromUint,
+  VOTE_ABSENT,
+  VOTE_NAY,
+  VOTE_YEA,
+} from '../vote-types'
 import VoteSummary from './VoteSummary'
 import VoteStatus from './VoteStatus'
 
 class VotePanelContent extends React.Component {
+  static propTypes = {
+    app: PropTypes.object.isRequired,
+  }
+  state = {
+    userBalance: null,
+    voterState: null,
+  }
+  componentDidMount() {
+    const { app, user, vote } = this.props
+    app
+      .call('getVoterState', vote.id, user)
```

Comment:
@onbjerg This doesn't seem to be working :(

Reply:
Just leaving this here for note: we should check if `vote.id` even exists (it's not set in `marshallVote`)